docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+datastore:
```

Comment:
The datastore service should have a unique name such as form-editor-datastaore so it is distinguishable in the overall docker-compose file for the entire system.